web/src/components/PartProfile/PartProfile.js

```diff
@@ -155,7 +157,10 @@ const PartProfile = ({
           {!isEditable && (
             <>
               <div className="h-px bg-indigo-200 mt-8" />
-              <h3 className="text-indigo-800 text-lg font-roboto tracking-wider mb-4">
+              <h3
+                className="text-indigo-800 text-lg font-roboto tracking-wider mb-4"
+                id="section"
```

Comment:
Could we make this id less generic? `id="comment-section"`

As much as possible we want our code to express intent, that way it will be easier to figure out what's going on later, or when someone else is reading it and `document.getElementById('section').scrollIntoView()` doesn't tell us much besides that it's scrolling somewhere. though `document.getElementById('comment-section').scrollIntoView()` tells us the intention nicely.

Reply:
my bad